### PR TITLE
28 feb

### PR DESCRIPTION
**Pull request by MarianBastiurea**
easy class using function for ArrayList

### CODE REVIEW

File: src/main/java/org/example/c06/ArrayList/Compare.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        }
+        System.out.println("second ArrayList " + numbers1 + " ");
+        for (int i = 10; i < 20; i++) {
+            numbers2.add(i);
+        }
+        System.out.println("third ArrayList " + numbers2 + " ");
+        boolean result=true;
+        boolean result1=true;
+        System.out.println(result);
+        if (result ==numbers1.equals(numbers)){
```
Comment (andreiursudev) [inline src/main/java/org/example/c06/ArrayList/Compare.java]: Simplify to `if (numbers1.equals(numbers)){`

File: src/main/java/org/example/c06/ArrayList/Compare.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            System.out.println("ArrayList numbers and number1 are equal");
+        }
+        if (result1 !=numbers1.equals(numbers2)){
```
Comment (andreiursudev) [inline src/main/java/org/example/c06/ArrayList/Compare.java]: Simplify to `if (!numbers1.equals(numbers2))`

File: src/main/java/org/example/c06/ArrayList/CopyArrayList.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package org.example.c06.ArrayList;
+
+import java.util.ArrayList;
+
+public class CopyArrayList {
+    public static void main(String[] args){
+        ArrayList<Integer> numbers=new ArrayList<>();
+        ArrayList<Integer> numbers1=new ArrayList<>();
```
Comment (andreiursudev) [inline src/main/java/org/example/c06/ArrayList/CopyArrayList.java]: Format code.

File: src/main/java/org/example/c06/ArrayList/IsEmpty.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ */
+public class IsEmpty {
+    public static void main(String[] args) {
+        ArrayList<Integer> numbers = new ArrayList<>();
+        ArrayList<Integer> numbers1 = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            numbers.add(i);
+        }
+        System.out.println("ArrayList is " + numbers + " ");
+        int size=numbers.size();
```
Comment (andreiursudev) [inline src/main/java/org/example/c06/ArrayList/IsEmpty.java]: Format code.

**Review by andreiursudev — APPROVED** [review]

File: src/main/java/org/example/c06/ArrayList/Compare.java (hunk 1 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c06/ArrayList/Compare.java]: done

Comment (MarianBastiurea) [reply src/main/java/org/example/c06/ArrayList/Compare.java]: done

File: src/main/java/org/example/c06/ArrayList/Compare.java (hunk 2 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c06/ArrayList/Compare.java]: done

File: src/main/java/org/example/c06/ArrayList/CopyArrayList.java (hunk 3 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c06/ArrayList/CopyArrayList.java]: done

File: src/main/java/org/example/c06/ArrayList/IsEmpty.java (hunk 4 above)
Comment (MarianBastiurea) [reply src/main/java/org/example/c06/ArrayList/IsEmpty.java]: done

**Pull request merged**